CHANGELOG.md

```diff
@@ -26,7 +30,9 @@
 A large refactor and simplification of the SDK moving most of the functionality into integrations used with
 `@sentry/browser` and `@sentry/node`.
 
-- Session tracking data sent by default. See our [release health docs for more details](https://docs.sentry.io/product/releases/health/). You can opt out of this behaviour by setting `autoSessionTracking: false` during SDK initialization.
+- Session tracking data sent by default. See our
+  [release health docs for more details](https://docs.sentry.io/product/releases/health/). You can opt out of this
```

Comment:
formatting change?

Reply:
Yep. Prettier keeps correcting this and I keep reverting it but decided this time to just commit the changes!